app/external_messaging/telegram.py

```diff
@@ -1,3 +1,18 @@
-# TODO: implement a concrete implementation in the future
-def post_telegram_message(account: str, msg: str):
-    print(f"[TelegramBot - {account}]: {msg}")
+# TODO: sample Python code that sends a text message to a user on Telegram using the python-telegram-bot library
+
+'''
+Make sure to replace YOUR_BOT_TOKEN_HERE with your bot token 
+and USER_CHAT_ID_HERE with the chat ID of the user you want to send the message to.
+You can obtain a bot token by creating a new bot with BotFather on Telegram.
+'''
+
+# function send_message which takes in two arguments: text and chat_id, 
+# sends the text message to the specified user on Telegram
+from telegram import Bot
+
+def send_message(text: str, chat_id: str):
+    bot = Bot(token='YOUR_BOT_TOKEN_HERE')
```

Comment:
The token shouldn't be hardcoded. I would recommend reading it from an environmental variable. To get a token from the environment you can use the [os.getenv](https://www.geeksforgeeks.org/python-os-getenv-method/) method. Remember to set the environment variable first for testing purposes.
Here's a tutorial for doing it on Windows: https://docs.oracle.com/en/database/oracle/machine-learning/oml4r/1.5.1/oread/creating-and-modifying-environment-variables-on-windows.html

If you're using MacOS or Linux you can type into the terminal the following command:
`export TELEGRAM_TOKEN="YOUR_BOT_TOKEN_HERE"`. You can access the token with `os.getenv("TELEGRAM_TOKEN")` in python.

Reply:
me absolutely agree, that the token should not be hardcoded, and it's much better to read it from an environmental variable.
 I appreciate you taking the time to provide me with instructions on how to set the environmental variable and access it. I will make sure to update the code accordingly and follow the tutorial you shared to set the environmental variable.

![meow learn](https://media.giphy.com/media/13HBDT4QSTpveU/giphy.gif)